From 1420bdea85caef6f7267f36ce6d968dccde2f21d Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 23 Nov 2016 10:20:09 +0200
Subject: [PATCH 0611/1345] net: mvpp2x: netmap: limit num_cos_queues to 1

commit  4ed54572eba7eeabe80b6a6de94b8698f3ef5aad from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- In netmap mode, limit the number of CoS queues to
  default 1 queue

Change-Id: I12c936fe9157225433582f19bc91bbebdf8efbe9
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33974
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
index 80664aa..ba5f52e 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
@@ -72,7 +72,11 @@
 #define CPN110_ADDRESS_SPACE_SIZE (16*1024*1024)
 
 /* Declaractions */
+#if defined(CONFIG_NETMAP) || defined(CONFIG_NETMAP_MODULE)
+u8 mv_pp2x_num_cos_queues = 1;
+#else
 u8 mv_pp2x_num_cos_queues = 4;
+#endif
 static u8 mv_pp2x_queue_mode = MVPP2_QDIST_SINGLE_MODE;
 static u8 rss_mode;
 static u8 default_cpu;
-- 
1.7.9.5

